src/MathOptInterface.jl

```diff
@@ -8,6 +8,9 @@ Abstract supertype for objects that implement the "Model" interface for defining
 an optimization problem.
 """
 abstract type ModelLike end
+@static if VERSION >= v"0.7-"
```

Comment:
Add comments explaining what this does. Are there any other MOI types we should define this for?

Reply:
Yes, I just started with one to see if every one is on the same page